composer.json

```diff
@@ -27,7 +27,8 @@
         "jackalope/jackalope-doctrine-dbal": "~1.2",
         "phpunit/phpunit": "~6.3",
         "ruflin/elastica": "~1.0",
-        "symfony/property-access": ">=2.3"
```

Comment:
The  `composer.lock` update is missing.

Reply:
Added

Reply:
oops, it's actually an error from me, sorry it should not be here!

Reply:
Right, I removed it